web/src/scss/variables.scss

```diff
@@ -35,3 +35,44 @@ $link-hover-color: #326de6;
 // misc
 $gitops-color: #e9e5ef;
 $border-radius: 4px;
+
+// custom branding fallbacks
+:root {
+  // navigation
+  --nav-color: $nav-color;
+  --nav-text-color: $nav-text-color;
+  --sub-nav-color: $sub-nav-color;
+  --sub-nav-text-color: $sub-nav-text-color;
+
+  // colors
+  --primary-color: var(--override-blue, $primary-color);
+  --secondary-color: var(--override-dark-gray, $secondary-color);
+  --accent-color: var(--override-blue, $accent-color);
+  --light-accent: $light-accent;
+  --muted-light-accent: $muted-light-accent;
+  --really-light-accent: $really-light-accent;
+  --light-taupe: $light-taupe;
+  --border-color: $border-color;
+
+  // errors and warnings
+  --error-color: $error-color;
+  --error-color-light: $error-color-light;
+  --warning-color: $warning-color;
+  --success-color: $success-color;
+
+  // text
+  --text-color-primary: $text-color-primary;
+  --text-color-secondary: $text-color-secondary;
+  --text-color-header: $text-color-header;
+  --text-color-body-copy: $text-color-body-copy;
+  --text-color-accent: $text-color-accent;
+  --text-color-info: $text-color-info;
+
+  // links
+  --link-color: var(--override-blue, $link-color);
+  --link-hover-color: var(--override-blue, $link-hover-color);
+
+  // misc
+  --gitops-color: $gitops-color;
+  --border-radius: $border-radius;
+}
```

Comment:
by default, these will all use the sass variable values. if a vendor wants to overide one of them they can just pass in a css variable and it should just override this. 

i added some nested overrides to mess around with. i suspect we will remove the ones like `--override-blue`